src/main/java/api/bpartners/annotator/repository/model/ExportTask.java

```diff
@@ -0,0 +1,76 @@
+package api.bpartners.annotator.repository.model;
+
+import static api.bpartners.annotator.repository.model.ExportTaskStatus.HealthStatus.SUCCEEDED;
+import static api.bpartners.annotator.repository.model.ExportTaskStatus.HealthStatus.UNKNOWN;
+import static api.bpartners.annotator.repository.model.ExportTaskStatus.ProgressionStatus.FINISHED;
+import static api.bpartners.annotator.repository.model.ExportTaskStatus.ProgressionStatus.PENDING;
+import static jakarta.persistence.CascadeType.ALL;
+import static jakarta.persistence.FetchType.EAGER;
+import static java.time.Instant.now;
+import static java.util.Comparator.comparing;
+import static java.util.Comparator.naturalOrder;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+@Entity
+@Table(name = "\"export_task\"")
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+@Data
+public class ExportTask {
+  @Id private String id;
+  private String jobId;
+  @CreationTimestamp private Instant submissionInstant;
+
+  @OneToMany(cascade = ALL, mappedBy = "exportTaskId")
+  private List<AnnotationBatch> annotationBatches;
+
+  @OneToMany(cascade = ALL, mappedBy = "taskId", fetch = EAGER)
+  @Builder.Default
+  private List<ExportTaskStatus> statusHistory = new ArrayList<>();
+
+  public ExportTaskStatus getStatus() {
+    return statusHistory.isEmpty()
+        ? ExportTaskStatus.builder()
+            .taskId(this.jobId)
+            .progression(PENDING)
+            .health(UNKNOWN)
+            .creationDatetime(now())
+            .build()
+        : statusHistory.stream()
+            .sorted(comparing(ExportTaskStatus::getCreationDatetime, naturalOrder()).reversed())
+            .toList()
+            .getFirst();
+  }
+
+  public void hasNewStatus(ExportTaskStatus status) {
```

Comment:
Why is this specific to export? Ready the code, I don't see anything that make it specific to export.